.gitpod.yml

```diff
@@ -4,8 +4,6 @@ image:
 tasks:
   - init: |
       yarn
-      nvm install $(cat .nvmrc)
-      nvm use
```

Comment:
Just because we are using a LTS version doesn't mean we will want to switch to it the second it is released. It would required testing prior to deciding to switch. 

We include the `.nvmrc` in order to restrict (or recommend) the desired version of Node. 

Yes, let's use the LTS version of the image, but let's not remove this section.

Reply:
👍